Test/GetMinuteQuarterTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once "./src/MyClass.php";
+class GetMinuteQuarterTest extends TestCase
+{
+    public $obj;
+
+    
+    /**
+     * @dataProvider positiveDataProvider
+     */
+    public function testPositive($minute, $expected)
```

Comment:
Taking into account you are using scalar type hints in `MyClass`, why don't you use it here?